tidb-lightning/tidb-lightning-compatibility-and-scenarios.md

```diff
@@ -0,0 +1,66 @@
+
+During data recovery, the snapshot data of the table is restored. See [Restore a table](/br/br-snapshot-manual.md#restore-a-table) for the procedure.
+
+#### #### Scenario 3: after the physical import is finished, perform DML operations to the table (not supported)
+
+In this scenario, you can only choose either [full snapshot backup](/br/br-snapshot-guide.md) or [log backup](/br/br-pitr-guide.md) for the backup operation of this table. You cannot back up and restore the full snapshot data and log backup data of this table.
+
+### Used with TiCDC
+
+This scenario is not compatible in the short term, because TiCDC can hardly keep up with the write speed of TiDB Lightning physical import mode, resulting in increasing cluster replication latency.
+
+You can perform in different scenarios as follows:
+
+- Scenario 1: the table does not need to be replicated downstream by TiCDC. In this scenario, the compatibility check will report an error after starting TiDB Lightning because TiCDC changefeed is enabled. If you are sure that these tables do not need to be replicated by TiCDC, you can change the `Lightning.check-requirements` parameter in the [TiDB Lightning configuration file](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-task) to `false` and then restart the task.
+
+- Scenario 2: The table needs to be replicated downstream by TiCDC. In this scenario, the compatibility check will report an error after starting TiDB Lightning because TiCDC changefeed is enabled. You need to change the `Lightning.check-requirements` parameter in the [TiDB Lightning configuration file](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-task) in the upstream TiDB cluster to `false`and then restart the task. After that, use TiDB Lightning to import a copy of the same data in the downstream TiDB cluster. If you have databases such as Redshift and Snowflake downstream, you can have these databases read CSV or Parquet files from a cloud storage service and write them to the database.
+
+## Scenarios for `IMPORT INTO`
+
+This section describes how to use `IMPORT INTO` together with [log backup](/br/br-pitr-guide.md) and [TiCDC](/ticdc/ticdc-overview.md).
+
+### Used with log backup
+
+In this scenario, the compatibility check will report an error after you submit the `IMPORT INTO` SQL because [PITR](/br/br-log-architecture.md#pitr) is enabled. If you are sure that these tables do not need to be backed up or backed up by [log backups](/br/br-pitr-guide.md), you can include `DISABLE_PRECHECK` (introduced in v8.0.0) in [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) of that SQL, and then resubmit it, so that the data import task ignores the compatibility check, and imports the data directly.
```

Comment:
```suggestion
In this scenario, the compatibility check will report an error after you submit the `IMPORT INTO` SQL because [PITR](/br/br-log-architecture.md#process-of-pitr) is enabled. If you are sure that these tables do not need to be backed up or backed up by [log backups](/br/br-pitr-guide.md), you can include `DISABLE_PRECHECK` (introduced in v8.0.0) in [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) of that SQL, and then resubmit it, so that the data import task ignores the compatibility check, and imports the data directly.
```